src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -782,11 +800,19 @@ You can opt to place this file in `$PROACTIVE_HOME/config/authentication/azure.c
  In case the parameter is set, several conditions have to be met:
  ** the deployment Scale Set template (to be specified in *armTemplateUrl*) should contain only a creation of next Azure resources inside Resource Group: the Scale Set and the Storage Account (no Azure Load Balancer, IP address, etc.)
  ** the target Azure Virtual Network should be provided by the user in the *targetNetwork* parameter.
+
+The next fields are related to the node configuration.
+
 - *rmUrl:* Optional parameter. Provide your server's URL or public IP (e.g. try.activeeon.com). If left unset, the default Resource Manager URL will be fetched from the system properties.
 - *rmCredentials:* Optional parameter. Provide credentials to connect to the Resource Manager. If left blank, the credentials of the logged user will be used.
 - *deploymentTimeout:* Delay before the Resource Manager can declare a deploying node as lost. To be set in _minutes_. You can set it to a reasonable time after which you expect the node to be running or marked as lost otherwise (e.g. _10_ minutes). Default value is _120_.
-- *cleaningDelay:* Periodical cleaning delay, expressed in _seconds_. Every _X_ seconds the Resource Manager will remove from the Scale Set any VM that has been identified as idle. Default value is _60_. Note that setting a very short interval implies more frequent communication between the Resource Manager and Azure, which might impact on performance or responsiveness.
-- *externalStorageAccount:* Optional parameter. Use it if you want to define a specific location of your ProActive node.jar worker; leave blank otherwise. If provided, the format should be: _{storageAccount}:{sas_key_token}[:has_node.jar]_.
+- *nodePreCommand:* Optional parameter. This multi-line field can be used to provide specific commands to execute right before launching the ProActive's node.jar application. The main difference with the previous user custom script is that _nodePreCommand_ will be run as _the same_ user as node.jar. This field is thus useful for operations that require matching users, e.g. mounting a shared volume in Windows or creating non-root paths/files in Linux. Please note the OS-specific conditions for these commands:
```

Comment:
-`right before` sounds a bit oral. I would suggest just `before`
-  `Contrary to the previous user custom script, _nodePreCommand_ will be run as _the same_ user as node.jar`